packages/confusing-browser-globals/test.js

```diff
@@ -4,7 +4,7 @@ let assert = require('assert');
 let globals = require('./');
 
 it('should return an Array of globals', function() {
-  assert.strictEqual(globals.constructor, Array);
+  assert.strictEqual(Array.isArray(globals), true);
```

Comment:
I just mean that instead of calling `mocha` you could call `jest` (and add it to dependencies).
And replace assertions with Jest syntax:

```js
expect(Array.isArray(globals)).toBe(true);
```